machine-learning/sentiment-analysis/sentiment-analysis.dig

```diff
@@ -0,0 +1,43 @@
+_export:
+  !include : config/params.yml
+  td:
+    database: ${dbname}
+    engine: hive
+
++prepare:
+  +shuffle:
+    _parallel: true
+
+    +train:
+      td>: queries/shuffle.sql
+      source: movie_review_train
+      create_table: movie_review_train_shuffled
+
+    +test:
+      td>: queries/shuffle.sql
+      source: movie_review_test
+      create_table: movie_review_test_shuffled
+
++train:
+  py>: sentiment.run
+  docker:
+    # Note: This image is temporary. It might change in the near future.
+    image: 'chezou/td-ml-base:latest'
+  _env:
+    TD_API_KEY: ${secret:apikey}
+    TD_API_SERVER: ${secret:endpoint}
+    AWS_ACCESS_KEY_ID: ${secret:aws_access_key_id}
+    AWS_SECRET_ACCESS_KEY: ${secret:aws_secret_access_key}
+    S3_BUCKET: ${secret:s3_bucket}
+
+#+predict:
```

Comment:
explain what predict does.

Why is this commented out?

Reply:
Prediction code works locally, but it doesn't work on k8s with current memory size limitation.

Reply:
Without prediction, just updating pre-trained model in training. 

Is this a good example for our customer ? I think it's better to be WIP until OOM issue is fixed.